go.mod

```diff
@@ -1 +1,5 @@
 module github.com/alexandrevicenzi/go-sse
+
+go 1.12
```

Comment:
go 1.11 should be here I think, otherwise may not be compatible.

Reply:
Adding support for go 1.12 in travis-ci should work.